src/util/util.ts

```diff
@@ -160,6 +160,24 @@ export function isRightSide(side: Side, isRTL: boolean, defaultRight: boolean =
   }
 }
 
+/**
+ * @hidden
+ * Given a side, return if it is the start
+ * based on the value of dir
+ * @param side the side
+ * @param isRTL whether the application dir is rtl
+ * @param defaultStart whether the default side is start
+ */
+export function isStartSide(side: Side, isRTL: boolean, defaultStart: boolean = true): boolean {
```

Comment:
can't we reuse isRightSide(side: Side, isRTL: boolean, defaultRight: boolean = false)?